tests/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Properties/VisualBasic/MapDynamicEnumValuesProviderTests.cs

```diff
@@ -122,7 +124,7 @@ private static IEnumerable<PageEnumValue> CreateEnumValueInstances(List<Tuple<st
             }
         }
 
-        private Dictionary<string, IEnumValue> CreateEnumValueMap(List<string> keys, IEnumerable<PageEnumValue> pageEnumValues)
+        private static Dictionary<string, IEnumValue> CreateEnumValueMap(List<string> keys, IEnumerable<PageEnumValue> pageEnumValues)
```

Comment:
Roslyn clearly didn't find this one?

Reply:
I'll see if I can create a minimal repro.

Reply:
Does Roslyn actually have this feature? I cannot get it to suggest converting a method to static on even a simple case:

```c#
class Program
{
    static void Main()
    {
        new Program().M();
    }

    private void M()
    {
        N();
    }

    private void N()
    {
    }
}
```

Reply:
It's part of roslyn-analyzers so you need that referenced. It definitely has it though, as I was getting it erroneously last night :)

Reply:
Seems it's related to being called from an xUnit fact: https://github.com/dotnet/roslyn-analyzers/issues/3019